.travis/validate-package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Alternatively, instead of putting this in `.travis`, we might also put it in `test/deployment` or so and make it part of the test suite.

Reply:
I'm also a fan of making this part of the test suite. Would write it in JS then though.

Reply:
> Would write it in JS then though.

Nah, the point is precisely that it is an external system running it, I'd say.

Reply:
Can still call an external process using javascript. (I'm mostly thinking that this might not work on my machine otherwise :D ).

Reply:
Ok, let's stick with the current place for now, and deprecate it once we have the more extensive JS version.

Reply:
Ah that last thing is a good point indeed. No bash on Windows at all?

Reply:
There are probably ways around it, would have to test, but I don't think it's just going to work out of the box.